### PR TITLE
small typos

### PR DESCRIPTION
**Pull request by bartocc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This addon provides a ~~polyfill~~ high fidelity reference implementation of
 [RFC #389](https://github.com/emberjs/rfcs/pull/389), including the proposed
 amendments in [RFC PR #620](https://github.com/emberjs/rfcs/pull/620).
 
-Please note that the while [RFC #389](https://github.com/emberjs/rfcs/pull/389)
+Please note that while the [RFC #389](https://github.com/emberjs/rfcs/pull/389)
```
Comment (chancancode) [inline README.md:12]:
```suggestion
Please note that while [RFC #389](https://github.com/emberjs/rfcs/pull/389)
```

File: README.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ This implementation has the following known limitations:
 
 * In Ember versions before 3.11, modifiers cannot be passed to the element,
   even with addons such as the [modifier manager](https://github.com/ember-polyfills/ember-modifier-manager-polyfill)
-  and [on modifier](https://github.com/buschtoens/ember-on-modifier) polyfills
-  are used. Doing so requires [RFC #435](https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md)
+  and the [on modifier](https://github.com/buschtoens/ember-on-modifier) polyfills.
+  Doing so requires [RFC #435](https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md)
```
Comment (chancancode) [inline README.md:51]: I think the actual typo here is "even with addons such as" -> "even **when** addons such as", the the rest of the original sentence makes sense (to me... I think?)

Comment (bartocc) [reply README.md]: I agree @chancancode 👍

I've updated the PR

**Comment by bartocc** [comment]
@chancancode anything else needed for this PR?

**Comment by chancancode** [comment]
@bartocc nope, thank you!

**Pull request merged**